gbpservice/nfp/configurator/agents/loadbalancer_v2.py

```diff
@@ -0,0 +1,978 @@
+            LOG.debug(msg)
+
+            method = getattr(self, "_%s" % (ev.id.lower()))
+            method(ev)
+        except Exception as err:
+            msg = ("Failed to perform the operation: %s. %s"
+                   % (ev.id, str(err).capitalize()))
+            LOG.error(msg)
+        finally:
+            if ev.id == lb_constants.EVENT_COLLECT_STATS:
+                """Do not say event done for collect stats as it is
+                   to be executed forever
+                """
+                pass
+            else:
+                msg = ("Calling event done for event=%s" % (ev.id))
+                LOG.info(msg)
+                self.sc.event_done(ev)
+
+    def _handle_event_loadbalancer(self, ev, operation):
+        data = ev.data
+        context = data['context']
+        loadbalancer = data['loadbalancer']
+        root_lb_id = self._root_loadbalancer_id('loadbalancer', loadbalancer)
+        agent_info = ev.data['context'].pop('agent_info')
+        service_vendor = agent_info['service_vendor']
+
+        try:
+            if operation == 'create':
+                driver_name = data['driver_name']
+                driver_id = driver_name + service_vendor
+                if (driver_id) not in self.drivers.keys():
+                    msg = ('No device driver on agent: %s.' % (driver_name))
+                    LOG.error(msg)
+                    self.plugin_rpc.update_status(
+                        'loadbalancer', loadbalancer['id'], root_lb_id,
+                        lb_constants.ERROR, lb_constants.OFFLINE, agent_info,
+                        None)
+                    return
+                driver = self.drivers[driver_id]
+                driver.load_balancer.create(context, loadbalancer)
+                LBaaSEventHandler.instance_mapping[loadbalancer['id']] \
+                    = driver_name
+            elif operation == 'update':
+                old_loadbalancer = data['old_loadbalancer']
+                driver = self._get_driver(service_vendor)
+                driver.load_balancer.update(context,
+                                            old_loadbalancer, loadbalancer)
+            elif operation == 'delete':
+                driver = self._get_driver(service_vendor)
+                driver.load_balancer.delete(context, loadbalancer)
+                del LBaaSEventHandler.instance_mapping[loadbalancer['id']]
+                return  # Don't update object status for delete operation
+        except Exception:
+            if operation == 'delete':
+                msg = (
+                    "Failed to delete loadbalancer %s" % (loadbalancer['id']))
+                LOG.warn(msg)
+                del LBaaSEventHandler.instance_mapping[loadbalancer['id']]
+            else:
+                self.plugin_rpc.update_status(
+                    'loadbalancer', loadbalancer['id'], root_lb_id,
+                    lb_constants.ERROR, lb_constants.OFFLINE,
+                    agent_info, None)
```

Comment:
@pkharat , if you look at config_orchestrator agent for [lbaasv1](https://github.com/oneconvergence/group-based-policy/blob/mitaka_21st_march_base/gbpservice/nfp/config_orchestrator/agent/loadbalancer.py#L378) and [lbaasv2](https://github.com/oneconvergence/group-based-policy/pull/358/files#diff-fd2defc07c5a3b6f7c451116a7afb3faR361), the update_status function only sent following info to neutron:
rpcClient.cctxt.cast(context, 'update_status',
                             obj_type=obj_type,
                             obj_id=obj_id,
                             status=status)

which means the objs included in the notification_date are not used.

That's why I set the obj to None here, to avoid sending useless date back and forth. For lbaasv2, a typical pool or member obj is a dict over 100k, which is a huge overhead.